src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -138,6 +138,26 @@ public void clickShowStats() {
 
     public void clickIndicateMissingResponses() {
         click(indicateMissingResponsesCheckbox);
+        expandPanels();
+    }
+
+    public void expandPanels() {
+        try {
+            // In case the panels are already expanded
```

Comment:
If the comment is not a full sentence, we usually start with a lower case letter